File: eniato/apps/accounts/domain/financial_institution.py
class FinancialInstitution:
    def __init__(self, attributes=None):
        attributes = attributes or {}
        self.id: int = attributes.get('id')
        self.code: int = attributes['code']
        self.name: str = attributes['name']
        self.logo: str = attributes['logo']

File: eniato/vue-app/src/apps/account/domain/FinancialInstitution.js
export class FinancialInstitution {
  constructor (attributes) {
    this._id = attributes.id
    this._code = attributes.code
    this._name = attributes.name
    this._logo = attributes.logo
  }

  get id () { return this._id }
  get code () { return this._code }
  get name () { return this._name }
  get logo () { return this._logo }

  toDTO () {
    return {
      id: this._id,
      code: this._code,
      name: this._name,
      logo: this._logo
    }
  }
}
